internal/metrics/metrics.go

```diff
@@ -127,3 +130,58 @@ func (s *ScheduleTimer) Mark(ctx context.Context, role, outcome string) {
 	metrics.FromContext(ctx).Histogram(
 		schedulingRetryTime, sparkRoleTag, outcomeTag, s.instanceGroupTag, s.retryTag).Update(now.Sub(s.lastSeenTime).Nanoseconds())
 }
+
+func ReportCrossZoneMetric(ctx context.Context, driverNodeName string, executorNodeNames []string, nodes []*v1.Node) {
+	executorNodesSet := make(map[string]interface{})
+	for _, n := range executorNodeNames {
+		executorNodesSet[n] = nil
+	}
+
+	zonesCounter := make(map[string]int64)
```

Comment:
can use `int`

Reply:
it's just the histogram expects int64, but I guess it's safe to convert int to int64 there